tests/unit_tests/data/test_aggregation.py

```diff
@@ -1804,7 +1804,7 @@ def test_update_timer_with_test_clock_sends_single_bar_to_handler(self):
             bid_size=Quantity.from_int(1),
             ask_size=Quantity.from_int(1),
             ts_event=1 * 60 * 1_000_000_000,  # 1 minute in nanoseconds
-            ts_init=1 * 60 * 1_000_000_000,  # 1 minute in nanoseconds
+            ts_init=1 * 60 * 1_000_000_000,
```

Comment:
There's a constant we might be able to use here. But, I think it's hard to access from Python, because of Cython.